src/main.rs

```diff
@@ -111,6 +126,14 @@ fn main() {
                 .help("Specify the language to output bindings in")
                 .possible_values(&["c++", "C++", "c", "C"]),
         )
+        .arg(
+            Arg::with_name("style")
+                .short("s")
+                .long("style")
+                .value_name("STYLE")
+                .help("Specify the language to output bindings in")
```

Comment:
I think you forgot to update the help here. Maybe "Specify the declaration style to use for bindings"?

Reply:
Done!